openstack/assets/dashboards/openstack_controller_screenboard.json

```
@@ -0,0 +1,713 @@
+{
+    "board_title": "OpenStack Controller - Overview",
```

```suggestion
    "board_title": "OpenStack Controller Overview",
```

Let's remove the `-` from the title. The dashboard url path is generated from this title and can be less friendly like this the jvm preset dashboard https://dddev.datadoghq.com/dash/integration/30/jvm---overview